src/sagemaker_training/mpi.py

```diff
@@ -95,11 +103,34 @@ def run(
 
         if wait:
             logger.info("Waiting for MPI process to finish.")
-            _wait_orted_process_to_finish()
+            gone, alive = _wait_orted_process_to_finish()
+            logger.info(f"Reporting status for ORTEd process. gone: {gone} alive: {alive}")
```

Comment:
what is gone and live for ORTE process here?

Reply:
https://psutil.readthedocs.io/en/latest/#psutil.wait_procs 
`(gone, alive)` tuple indicates which processes are gone and which ones are still alive.